core/templates/dev/head/App.ts

```diff
@@ -92,6 +92,10 @@ angular.module('oppia').config([
       $locationProvider.html5Mode(true);
     }
 
+    // The default location prefix is '!' we have to change it to '' to avoid
```

Comment:
Hi @seanlip
PTAL here, the default location provider prefix is changed to `'!'` we have to override the prefix with `''` to avoid ! in the URLs (happens in admin panel) see the 4th point in [this docs](https://docs.angularjs.org/guide/migration#migrating-from-1-5-to-1-6)

Reply:
Hi @NishealJ -- you didn't answer this question?

Reply:
Hm, yeah, that looks a bit weird.

I think it's your decision. I'm happy with anything that works. Maybe keeping it as it is now might be better for the time being, but that's just a preference and it's fine to override it as you see fit.

Reply:
Definitely, I'm overriding now it will be helpful for me later we canbuild the admin navbar with #! hrefs Thanks!